daemon/src/projection.rs

```diff
@@ -252,6 +255,11 @@ impl Aggregated {
             return Some(extract_payout_amount(tx, script));
         }
 
+        if let Some(tx) = self.refund_tx {
+            let script = self.latest_dlc?.script_pubkey_for(role);
+            return Some(extract_payout_amount(tx, script));
+        }
+
         let tx = self.cet.or(self.timelocked_cet)?;
         let dlc = self
```

Comment:
Could change below `.expect("dlc to be present when we have a cet");` to be just `?` as well I suppose.

Reply:
Done: https://github.com/itchysats/itchysats/pull/1894/commits/613cb3c6c4737c5df1403bc16602af75d2ddfb11.